### PR TITLE
add rainbond operator chart

### PR DESCRIPTION
**Pull request by barnettZQG**
**What this PR does / why we need it**:
add rainbond-operator chart to  `https://developer.aliyun.com/hub`

**If applicable**:
- [x] this PR contains documentation (是否包含文档)
- [x] this PR contains unit tests (是否包含测试、实际环境有效性验证)
- [x] this PR has been tested for backwards compatibility (是否向后兼容)

### CODE REVIEW

File: submitted/rainbond-operator/LICENSE [hunk 1]
```diff
@@ -0,0 +1,165 @@
+                   GNU LESSER GENERAL PUBLIC LICENSE
```
Comment (wonderflow) [inline submitted/rainbond-operator/LICENSE]: This License file should be removed as it conflicts with license of this project.

File: submitted/rainbond-operator/README.md [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+kubectl create ns rbd-system
+
+helm install rainbond-operator ./chart --namespace=rbd-system
+```
+
+## Installing ETCD Operator
+
+### Official Documentation
+
+Official project documentation found [here].
```
Comment (wonderflow) [inline submitted/rainbond-operator/README.md]: link?

File: submitted/rainbond-operator/LICENSE (hunk 1 above)
Comment (barnettZQG) [reply submitted/rainbond-operator/LICENSE]: Has been removed

File: submitted/rainbond-operator/README.md (hunk 2 above)
Comment (barnettZQG) [reply submitted/rainbond-operator/README.md]: The readme has been changed.

File: submitted/rainbond-operator/README.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+| `openapi.image.downloadURL`         | rainbond package download url                 | `Official download address` |
+| `openapi.image.downloadMD5`         | rainbond package SHA256                 | `Official package SHA256` |
+| `openapi.image.installMode`         | The installation mode, WithPackage or WithoutPackage     | `WithoutPackage` |
+| `openapi.image.imageRepository`     | The default image address              | `Oregistry.cn-hangzhou.aliyuncs.com/goodrain` |
+
+
+Specify each parameter using the `--set key=value[,key=value]` argument to `helm install`. For example:
+
+```bash
+$ helm install --name rainbond-operator --set image.tag=v1.0.0-beta2 ./chart
```
Comment (hongchaodeng) [inline submitted/rainbond-operator/README.md]: `helm install --name` is helm 2.0 syntax. 

3.0 syntax looks like:
```
helm install [NAME] [CHART] [flags]
```

Comment (barnettZQG) [reply submitted/rainbond-operator/README.md]: changed

**Pull request merged**